src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/WT/AltimeterIndicator.js

```diff
@@ -1503,9 +1503,21 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
     updateCursorScrolling(_altitude) {
         if (this.cursorIntegrals) {
             let hideZeros = (this.aircraft == Aircraft.A320_NEO) ? true : false;
-
-            this.cursorIntegrals[0].update(_altitude, 10000, (hideZeros) ? 10000 : undefined);
-            this.cursorIntegrals[1].update(_altitude, 1000, (hideZeros) ? 1000 : undefined);
+            // if(this.aircraft == Aircraft.CJ4){
+            //     if(_altitude < 10000){
+            //         this.cursorIntegrals[0].clear("X");

```

Comment:
Replacing the "X" with "▓" would recreate the correct altimeter behaviour but it seems like the sim does display unicode.

Reply:
@mattnischan